eslint fails on playwright-tests dir

```
$ make test
node lib/bin/enforce-node-version.js
npx eslint --cache --max-warnings 0 .

/odk/central-backend/oidc-dev/playwright-tests/playwright.config.js
  10:29  error  Unable to resolve path to module '@playwright/test'  import/no-unresolved
```

This may be correct, but not very helpful - lint should probably not need every sub-project's dependencies to install.

--- a/oidc-dev/fake-oidc-server/.eslintrc.cjs
+++ b/oidc-dev/fake-oidc-server/.eslintrc.cjs
@@ -7,14 +7,12 @@
 // including this file, may be copied, modified, propagated, or distributed
 // except according to the terms contained in the LICENSE file.
 
-const rules = {};
-
-// This rule does not work if the node_modules directory has not been populated.
-// If this rule is enabled here, `npm clean-install` will need to be run in this
-// directory before eslint is run.
-if (process.env.CI) rules['import/no-unresolved'] = 'off';
-
 module.exports = {
   extends: '../../.eslintrc.json',
-  rules,
+  rules: {
+    // This rule does not work if the node_modules directory has not been
+    // populated.  If this rule is enabled here, `npm clean-install` will need
+    // to be run in this directory before eslint is run.
+    'import/no-unresolved': 'off',
+  },
 };

--- a/oidc-dev/playwright-tests/.eslintrc.js
+++ b/oidc-dev/playwright-tests/.eslintrc.js
@@ -13,16 +13,9 @@ module.exports = {
     browser: true, // for page.waitForFunction() code
   },
   rules: {
-    // This rule does not work if the node_modules directory has not been populated.
-    // Downloading playwright is quite slow, so it's probably better we don't have
-    // to do that before linting.
+    // This rule does not work if the node_modules directory has not been
+    // populated.  Downloading playwright is quite slow, so it's probably better
+    // we don't have to do that before linting.
     'import/no-unresolved': 'off',
-  },
-  settings: {
-    'import/resolver': {
-      node: {
-        moduleDirectory: 'node_modules',
-      },
-    },
   },
 };

--- a/oidc-dev/playwright-tests/.eslintrc.js
+++ b/oidc-dev/playwright-tests/.eslintrc.js
@@ -7,11 +7,19 @@
 // including this file, may be copied, modified, propagated, or distributed
 // except according to the terms contained in the LICENSE file.
 
+const rules = {};
+
+// This rule does not work if the node_modules directory has not been populated.
+// Downloading playwright is quite slow, so it's probably better we don't have
+// to do that before linting.
+if (process.env.CI) rules['import/no-unresolved'] = 'off';
+
 module.exports = {
   extends: '../../.eslintrc.json',
   env: {
     browser: true, // for page.waitForFunction() code
   },
+  rules,
   settings: {
     'import/resolver': {
       node: {

--- a/oidc-dev/playwright-tests/.eslintrc.js
+++ b/oidc-dev/playwright-tests/.eslintrc.js
@@ -7,17 +7,16 @@
 // including this file, may be copied, modified, propagated, or distributed
 // except according to the terms contained in the LICENSE file.
 
-const rules = {};
-
-// This rule does not work if the node_modules directory has not been populated.
-// Downloading playwright is quite slow, so it's probably better we don't have
-// to do that before linting.
-if (process.env.CI) rules['import/no-unresolved'] = 'off';
-
 module.exports = {
   extends: '../../.eslintrc.json',
   env: {
     browser: true, // for page.waitForFunction() code
   },
-  rules,
+  settings: {
+    'import/resolver': {
+      node: {
+        moduleDirectory: 'node_modules',
+      },
+    },
+  },
 };

--- a/oidc-dev/playwright-tests/.eslintrc.js
+++ b/oidc-dev/playwright-tests/.eslintrc.js
@@ -7,19 +7,17 @@
 // including this file, may be copied, modified, propagated, or distributed
 // except according to the terms contained in the LICENSE file.
 
-const rules = {};
-
-// This rule does not work if the node_modules directory has not been populated.
-// Downloading playwright is quite slow, so it's probably better we don't have
-// to do that before linting.
-if (process.env.CI) rules['import/no-unresolved'] = 'off';
-
 module.exports = {
   extends: '../../.eslintrc.json',
   env: {
     browser: true, // for page.waitForFunction() code
   },
-  rules,
+  rules: {
+    // This rule does not work if the node_modules directory has not been populated.
+    // Downloading playwright is quite slow, so it's probably better we don't have
+    // to do that before linting.
+    'import/no-unresolved': 'off',
+  },
   settings: {
     'import/resolver': {
       node: {

